lib/core/presentation/pages/left_panel/left_panel_page.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:app/core/application/auth/auth_bloc.dart';
+import 'package:app/core/presentation/pages/left_panel/widgets/left_panel_profile_info.dart';
+import 'package:app/core/presentation/pages/left_panel/widgets/left_panel_tile_widget.dart';
+import 'package:app/core/presentation/widgets/back_button_widget.dart';
+import 'package:app/core/presentation/widgets/common/appbar/lemon_appbar_widget.dart';
+import 'package:app/core/presentation/widgets/theme_svg_icon_widget.dart';
+import 'package:app/core/utils/modal_utils.dart';
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/i18n/i18n.g.dart';
+import 'package:app/router/app_router.gr.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_vibrate/flutter_vibrate.dart';
+import 'package:slang/builder/utils/string_extensions.dart';
+
+@RoutePage()
+class LeftPanelPage extends StatelessWidget {
```

Comment:
What does LeftPanelPage means ? Doesn't belong to any feature ? Naming like this may cause confusing